src/language-xml/options.js

```diff
@@ -0,0 +1,26 @@
+"use strict";
+
+module.exports = {
+  xmlSelfClosingSpace: {
+    type: "boolean",
+    category: "XML",
+    default: true,
+    description: "Adds a space before self-closing tags.",
+  },
+  xmlWhitespaceSensitivity: {
+    type: "choice",
+    category: "XML",
+    default: "strict",
+    description: "How to handle whitespaces in XML.",
+    choices: [
+      {
+        value: "strict",
+        description: "Whitespaces are considered sensitive in all elements.",
+      },
+      {
+        value: "ignore",
+        description: "Whitespaces are considered insensitive in all elements.",
+      },
+    ],
+  },
```

Comment:
We need discussion on these options

/cc @prettier/core 

I think the `xmlSelfClosingSpace` option is unnecessary

Reply:
@evilebottnawi I don't think we need, it's still a plugin, we should let plugin decide. We can just ship it together with Prettier.

Reply:
@fisker SVG is part of web and we don't close this issue in favor plugin because we want to move it into core, integration HTML + SVG will be painful with plugins due architecture problems, fixes in plugins will be painful too

Reply:
> I think the xmlSelfClosingSpace option is unnecessary

I agree! That’s exactly the type of option Prettier should _not_ have. We don’t teams arguing over if they should pick true or false. We should always format like we do in HTML in this case.

Btw – I think it’s called “whitespace”, not “whitespaces”.